srcpkgs/eclipse/template

```diff
@@ -14,11 +14,11 @@ homepage="https://eclipse.org"
 nopie=yes
 make_dirs="/usr/lib/eclipse/dropins 0770 root users"
 
-_patch=R
+_patch=202306081200
 _edition=java
 _mirror="http://ftp-stud.fht-esslingen.de/pub/Mirrors/eclipse"
-distfiles="${_mirror}/technology/epp/downloads/release/${_release}/${_patch}/eclipse-${_edition}-${_release}-${_patch}-linux-gtk-x86_64.tar.gz"
-checksum=ccd07314798d6b152eb73321ff9a4e8252622a1e4d02f495c2f3d8c80e9156be
+distfiles="${_mirror}/technology/epp/downloads/release/${_release}/${_patch}/eclipse-${_edition}-${_release}-R-linux-gtk-x86_64.tar.gz"
```

Comment:
Looks like they moved file to match old pattern.

Reply:
Thanks, fixed!